playwright-e2e/tests/create-case.spec.ts

```diff
@@ -4,14 +4,16 @@ import { newSwanseaLocalAuthorityUserOne } from "../settings/userCredentials";
 import { CreateCase } from "../pages/create-case";
 import { StartApplication } from "../pages/start-application";
 import { OrdersAndDirectionSought } from "../pages/orders-and-directions";
+import {AllocationProposal} from "../pages/allocation-proposal";
```

Comment:
Hi @Braimah101 {AllocationProposal} should be { AllocationProposal } (spaces)

Reply:
True